vendor/github.com/containers/virtcontainers/hypervisor.go

```diff
@@ -109,6 +109,13 @@ type HypervisorConfig struct {
 	// ImagePath is the guest image host path.
 	ImagePath string
 
+	Unconstrained_sockets      uint32
```

Comment:
These variables should be documented (also it would be worth explaining what "unconstrained" means briefly).